### PR TITLE
Extract Apple Config processing functions

### PR DESCRIPTION
**Pull request by SlayterDev**
https://app.asana.com/0/72649045549333/1201640241287329/f

### CODE REVIEW

File: scripts/inject.mjs [hunk 1]
```diff
@@ -45,29 +47,40 @@ async function init() {
...
     const replaceString = "/* global contentScopeFeatures */";
-    const injectScript = await readFile(injectScriptPath);
-    const contentScope = await generateContentScope();
+    const injectScript = await rollupScript(injectScriptPath, `inject${platformName}`);
+    const contentScope = await rollupScript(contentScopePath, contentScopeName);
     const outputScript = injectScript.toString().replace(replaceString, contentScope.toString());
     console.log(outputScript);
 }
 
+async function initApple() {
```
Comment (jonathanKingston) [inline scripts/inject.mjs:68]: This function can be removed now it seems.

**Review by jonathanKingston — CHANGES_REQUESTED** [review]
Looks fine to me, one function to be removed.

Did you check the generated output of Chrome and Firefox, happy to check them over just a little nervous without some manual verification.

**Comment by SlayterDev** [comment]
@jonathanKingston It seems with the rollup it moves everything in those files to an anonymous function where there used to be some parts at the top level. Also some indentation is added. There shouldn't be any change to functionality but feel free to take a peak if you like.

**Comment by jonathanKingston** [comment]
As some files have changed here in the repo are you able to rebase and rebuild? We're good to merge then.

**Review by jonathanKingston — APPROVED** [review]

**Pull request merged**